src/site/content/en/blog/gpu/index.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Access modern GPU features with WebGPU
+subhead: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the web.
+authors:
+  - beaufortfrancois
+  - cwallez
+date: 2021-08-26
+#updated: 2021-08-26
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+thumbnail: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+description: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the web.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - games
+feedback:
+  - api
+stack_overflow_tag: webgpu
+---
+
+## What is WebGPU? {: #what }
+
+[WebGPU] is a new web API that exposes modern computer graphics capabilities,
+specifically Direct3D 12, Metal, and Vulkan, for performing rendering and
+computation operations on a graphics processing unit (GPU).
+
+<figure class="w-figure">
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/WHoJmX2IU7roV4iabH6M.png", alt="Architecture diagram showing WebGPUs connection between OS APIs and Direct3D 12, Metal, and Vulkan.", width="800", height="313" %}
+  <figcaption class="w-figcaption">WebGPU architecture diagram</figcaption>
+</figure>
+
+This goal is similar to the [WebGL] family of APIs, but WebGPU enables access to
+more advanced features of GPUs. Whereas WebGL is mostly for drawing images but
+can be repurposed with great effort for other kinds of computations, WebGPU
+provides first-class support for performing general computations on the GPU.
+
+After four years of development in the [W3C's "GPU for the Web" Community
+Group], WebGPU is now ready for developers to try in Chrome and give feedback on
+the API and the shading language.
+
+{% Blockquote 'Mr.doob, Creator of Three.js' %}
+"After a decade of WebGL bringing 3D graphics to the web and enabling all sorts
+of new experiences, it's now time to upgrade the stack and help web developers
+take full advantage of modern graphics cards. WebGPU arrives just in time!"
+{% endBlockquote %}
+
+{% Blockquote 'David Catuhe, Creator of Babylon.js' %}
+WebGPU gets us closer to the metal and it also unlocks the power of compute
+shader for Web developers. New 3D experiences can be built today on [Babylon.js
+Playground].
+{% endBlockquote %}
+
+{% Video src="video/vvhSqZboQoZZN9wBvoXq72wzGAf1/Xb7LvsJ5e8efTssp94c6.mov", autoplay=true, muted=true, playsinline=true, loop=true %}
```

Comment:
Maybe we could have an explanation of what we see in the video: A Babylon.js demo <link> of a rough sea being simulated using WebGPU's compute shader capability.

Reply:
Good catch! Done.